layouts/partials/hooks/body-end.html

```diff
@@ -0,0 +1,9 @@
+<!-- scripts for algolia docsearch -->
```

Comment:
I think we also need to stick this in a conditional because otherwise even if you set algolia_docsearch to false, if the scripts are here and you enable GCSE, you get the Algolia results from the search box rather than the GCSE ones.

I know the current file is empty but you could instead have the committed version look like

{{ with .Site.Params.algolia_docsearch  }}
&gt;!-- scripts for algolia docsearch --&lt;
{{ end }}

Does that make sense in terms of user experience? I don't want to go too bananas with conditionals but at the same time we don't want people getting unintuitive results. Also if you accidentally enable both in config.toml, the Algolia one currently wins if it has scripts there. We may need to do some rewording in the docs.